MediaGalleryUi/view/adminhtml/web/js/image/image-edit.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+define([
+    'jquery',
+    'underscore',
+    'uiComponent',
+    'Magento_MediaGalleryUi/js/action/getDetails'
+], function ($, _, Component, getDetails) {
+    'use strict';
+
+    return Component.extend({
+        defaults: {
+            template: 'Magento_MediaGalleryUi/image/image-edit',
+            modalSelector: '',
+            imageEditDetailsUrl: '/media_gallery/image/details',
+            saveDetailsUrl: '/media_gallery/image/saveDetails',
+            images: [],
+            image: null,
+            modules: {
+                mediaGridMessages: '${ $.mediaGridMessages }'
+            }
+        },
+
+        /**
+         * Init observable variables
+         *
+         * @return {Object}
+         */
+        initObservable: function () {
+            this._super()
+                .observe([
+                    'image'
+                ]);
+
+            return this;
+        },
+
+        /**
+         * Get image details by ID
+         *
+         * @param {String} imageId
+         */
+        showEditDetailsPanel: function (imageId) {
+            if (_.isUndefined(this.images[imageId])) {
+                getDetails(this.imageEditDetailsUrl, imageId).then(function (imageDetails) {
```

Comment:
getDetails is now working with multiple imageIds, please adjust the code accordingly - that should resolve the test failure
```suggestion
                getDetails(this.imageEditDetailsUrl, [imageId]).then(function (imageDetails) {
```